### PR TITLE
LPS-166689 Getting the current Form context language

### PR DESCRIPTION
**Pull request by rijema**
https://issues.liferay.com/browse/LPS-166689

### CODE REVIEW

**Comment by rijema** [comment]
ci:test:sf

**Comment by rijema** [comment]
ci:test:relevant

**Comment by rijema** [comment]
ci:test:relevant

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java [hunk 1]
```diff
@@ -91,9 +90,11 @@ public Map<String, Object> getParameters(
 		).build();
 	}
 
-	private int _getFirstDayOfWeek() {
+	private int _getFirstDayOfWeek(
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext) {
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java:94]: How about changing this parameter to be just Locale? Do you agree?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java [hunk 2]
```diff
@@ -60,12 +59,12 @@ public Map<String, Object> getParameters(
 		DDMFormFieldRenderingContext ddmFormFieldRenderingContext) {
 
 		return HashMapBuilder.<String, Object>put(
-			"firstDayOfWeek", _getFirstDayOfWeek()
+			"firstDayOfWeek", _getFirstDayOfWeek(ddmFormFieldRenderingContext)
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java:62]: Here, you could replace to: `dmFormFieldRenderingContext.getLocale()`.

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java [hunk 3]
```diff
@@ -91,9 +90,11 @@ public Map<String, Object> getParameters(
 		).build();
 	}
 
-	private int _getFirstDayOfWeek() {
+	private int _getFirstDayOfWeek(
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext) {
+
 		WeekFields weekFields = WeekFields.of(
-			LocaleThreadLocal.getThemeDisplayLocale());
+			ddmFormFieldRenderingContext.getLocale());
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java:97]: And here, you replace to the parameter `Locale locale` :+1:

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi @rijema ! Thanks for contributing :+1: 
I finished my review. Let me know if you have questions :smile:

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java (hunk 1 above)
Comment (rijema) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java]: Done!

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java (hunk 2 above)
Comment (rijema) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java]: Done

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java (hunk 3 above)
Comment (rijema) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java]: Done

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java (hunk 2 above)
Comment (rijema) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/java/com/liferay/dynamic/data/mapping/form/field/type/internal/date/DateDDMFormFieldTemplateContextContributor.java]: Done

**Comment by rijema** [comment]
Hi @marcelabc , so sorry for the delay in answering your requests. I just updated the PR with the changes. Lemme know if anything else is necessary. Regards!

**Comment by rijema** [comment]
ci:test:relevant

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by rijema** [comment]
Hi @marcelabc , do I have to manually forward this ticket as well?

**Comment by marcelabc** [comment]
No, because it has passed in CI :smile: :tada:

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**